datamanagement/query_gsc_for_all_dlp_fastqs.py

```diff
@@ -36,35 +37,39 @@ def main(storage_name, all, tag_name=None):
 
     else:
         for sequencing in sequencing_list_all:
-            if sequencing['dlpsequencingdetail']:
-                if sequencing['dlpsequencingdetail']['number_of_lanes_requested'] != len(sequencing['dlplane_set']):
-                    sequencing_list.append(sequencing)
+            if sequencing['number_of_lanes_requested'] != len(sequencing['dlplane_set']):
+                sequencing_list.append(sequencing)
 
     for sequencing in sequencing_list:
 
         # Query GSC for FastQs
-        import_info = import_gsc_dlp_paired_fastqs(
-            colossus_api,
-            tantalus_api,
-            sequencing["library"],
-            storage,
-            tag_name)
+        try:
+            import_info = import_gsc_dlp_paired_fastqs(
+                colossus_api,
+                tantalus_api,
+                sequencing["library"],
+                storage,
+                tag_name,
+                dry_run=dry_run)
+        except Exception as e:
+            print("Library {} failed to import: {}".format(sequencing["library"], e))
+            continue
 
         if import_info is None:
             continue
 
         # Re-get the sequencing details, may be unnecessary but safer
         # given that it is nested in sequencing and may have been changed
         # in a previous iteration of this loop
-        sequencingdetails = colossus_api.get('sequencingdetails', id=sequencing['dlpsequencingdetail']['id'])
+        sequencingdetails = colossus_api.get('sequencing', id=sequencing['id'])
 
         if sequencingdetails['gsc_library_id'] is not None:
             if sequencingdetails['gsc_library_id'] != import_info['gsc_library_id']:
-                raise Exception('gsc library id mismatch in dlpsequencingdetail {} '.format(sequencingdetails['id']))
+                raise Exception('gsc library id mismatch in sequencing {} '.format(sequencingdetails['id']))
```

Comment:
still referencing sequencingdetails, perhaps do a global search in sisypus to catch these

Reply:
oh, thats what the variable is called... rename the variable to just sequencing?

Reply:
Hmm we're already using sequencing when we iterate through sequencing_list. Would sequencing_info be more appropriate?